### ISSUE TITLE
Cannot find ... when opening asset

### ISSUE THREAD

**Issue by stefnotch** [FlaxEngine/FlaxEngine#394]
**Issue description:**
I have a project where I procedurally generate textures and save them as assets. Then I open them in Flax. This sometimes results in the following warnings

```
[ 00:00:45.002 ]: [Info] Shader 'FlaxEngine.Material: a139903749af4ec84fdc64b3aff00153, 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Content/Old/RenderSDFMaterial 0.flax', Refs: 2' compiled! Cache size: 1452 bytes
[ 00:00:45.002 ]: [Info] Start searching asset with ID: 3a0165ee49085120e10245b136f55970 in 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Content'. 21 potential files to check...
[ 00:00:45.005 ]: [Info] Start searching asset with ID: 3a0165ee49085120e10245b136f55970 in 'C:\Users/Stefnotch/Documents/GitHub/Flax/FlaxEngine/Content'. 153 potential files to check...
[ 00:00:45.017 ]: [Warning] Cannot find 3a0165ee49085120e10245b136f55970.
[ 00:00:45.017 ]: [Warning] Invalid asset ID (3a0165ee49085120e10245b136f55970).
[ 00:00:45.017 ]: [Warning] Cannot perform workspace scan for '05b4226d4b7bba4cc3197a8dde67c9b8'. Too often call. Time diff: 15 ms
[ 00:00:45.017 ]: [Warning] Invalid asset ID (05b4226d4b7bba4cc3197a8dde67c9b8).
[ 00:00:45.816 ]: [Info] Creating package at 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails/cache_034e41220be5428e828aed01e5069f15.flax'. Silent Mode: true
[ 00:00:45.816 ]: [Info] File 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails/cache_034e41220be5428e828aed01e5069f15.flax' is in use. Trying to release handle to it.
[ 00:00:57.471 ]: [Info] Start searching asset with ID: eeda8af64e681d79ce532aba2b185234 in 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Content'. 21 potential files to check...
[ 00:00:57.471 ]: [Info] Start searching asset with ID: eeda8af64e681d79ce532aba2b185234 in 'C:\Users/Stefnotch/Documents/GitHub/Flax/FlaxEngine/Content'. 153 potential files to check...
[ 00:00:57.482 ]: [Warning] Cannot find eeda8af64e681d79ce532aba2b185234.
[ 00:00:57.482 ]: [Warning] Invalid asset ID (eeda8af64e681d79ce532aba2b185234).
```

**Steps to reproduce:**
1. Clone https://github.com/stefnotch/anime-rendering-experiments
2. Open `Anime Rendering/Content/VectorAsset` in Flax
3. Click on `Generate Textures`
4. Open the generated `ColorTexture` or `DistanceTexture`
5. Recompile scripts
6. Repeat the steps above a few times
7. Check out the output log

![image](https://user-images.githubusercontent.com/10220080/112687343-a8d95580-8e77-11eb-9e12-d87ba3e0d303.png)

P.S. For those who are curious, `RenderSDFMaterial` renders a colored signed distance field. Two well made 64x64 textures and you got yourself a super sharp picture :)

**Flax version:**
Flax 1.1

**Comment by stefnotch** [comment]
After closing and reopening this project, the following warnings started popping up

```
[ 00:00:04.518 ]: [Info] Saving assets cache to 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/AssetsCache.dat', entries: 158
[ 00:00:04.668 ]: [Info] Creating package at 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails/cache_bd0857fbc1dd4016b5549480d1623b09.flax'. Silent Mode: true
[ 00:00:04.668 ]: [Warning] Win32::GetLastError() = 0x20
[ 00:00:04.668 ]: [Warning] Cannot open file 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails/cache_bd0857fbc1dd4016b5549480d1623b09.flax'
[ 00:00:04.668 ]: [Warning] Failed to save asset previews atlas 'FlaxEditor.PreviewsCache: 51d88991486a0bbe4b4d3c9f55a2d95a, 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails\cache_bd0857fbc1dd4016b5549480d1623b09.flax', Refs: 1'.
[ 00:00:05.738 ]: [Info] Creating package at 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails/cache_bd0857fbc1dd4016b5549480d1623b09.flax'. Silent Mode: true
[ 00:00:05.738 ]: [Warning] Win32::GetLastError() = 0x20
[ 00:00:05.738 ]: [Warning] Cannot open file 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails/cache_bd0857fbc1dd4016b5549480d1623b09.flax'
[ 00:00:05.738 ]: [Warning] Failed to save asset previews atlas 'FlaxEditor.PreviewsCache: 51d88991486a0bbe4b4d3c9f55a2d95a, 'C:\Users/Stefnotch/Documents/GitHub/Flax/anime-rendering/Cache/Thumbnails\cache_bd0857fbc1dd4016b5549480d1623b09.flax', Refs: 1'.
```

**Comment by mafiesto4** [comment]
Fixed by ebf2b0a2f3b0b3ab4723cfdd76f9273fab6a8216.
